pallets/subtensor/tests/staking.rs

```diff
@@ -1505,21 +1505,21 @@ fn test_full_with_delegating() {
         SubtensorModule::emit_inflation_through_hotkey_account(&hotkey1, 0, 1000);
         assert_eq!(
             SubtensorModule::get_stake_for_coldkey_and_hotkey(&coldkey0, &hotkey0),
-            601
-        ); // 200 + 1000 x ( 200 / 500 ) = 200 + 400 = 600 ~= 601
+            660
+        ); // 200 + 1000 x ( 200 / 500 )+ 60 = 200 + 400  ~= 660
```

Comment:
pls define a function to calculate it. From function name, parameter name. we can understand what's the formula

Reply:
I agree this is needed, but this is somewhat significant effort because of rounding.

Reply:
660 is a correct value. The formula here is
```
validator_take = take * validator_emission = 10% * 1000 = 100
old_stake + (validator_emission - validator_take) * stake_for_coldkey_and_hotkey / total_stake_for_hotkey + validator_take
=
200 + 900 * 200 / 500 + 100 = 660
```

Reply:
While this is desirable , it touches a wider issue of how we verify assertions through out all our tests. I have created an issue for this , and thing it should be handled outside this PR. 

https://github.com/orgs/opentensor/projects/27/views/9?pane=issue&itemId=63940578